### ISSUE TITLE
Issue on page /example_ml_iris.html

### ISSUE THREAD

**Issue by olivertomic** [vaexio/vaex#1058]
Thanks a lot for vaex, it is a great package. 

In the example code

df.export("shuffled", shuffle=True)
df = vaex.open("shuffled.hdf5)
df_train, df_test = df.ml.train_test_split(test_size=0.2)

I guess there are a couple of typos. I guess the correct code should be

df.export("shuffled.hdf5", shuffle=True) # The file name was missing .hdf5
df = vaex.open("shuffled.hdf5") # is a quote missing at the end of the string

**Comment by JovanVeljanoski** [comment]
Took a while, but it has been fixed. Thanks!
